pkg/replica/replica.go

```diff
@@ -988,6 +994,37 @@ func (r *Replica) readMetadata() (bool, error) {
 	return len(r.diskData) > 0, nil
 }
 
+func (r *Replica) tryRecoverVolumeMetaFile(head string) error {
+	filePath := r.diskPath(volumeMetaData)
+
+	err := r.unmarshalFile(filePath, &r.info)
+	// recover metadata file that does not exist or is empty
+	if !os.IsNotExist(err) && !errors.Is(err, io.EOF) {
+		return err
+	}
```

Comment:
Replace by `checkValidVolumeMetaData` and if not valid, try to do the following recovery.

Reply:
Replaced. Thanks.